igor-web/src/main/java/com/netflix/spinnaker/igor/artifacts/ArtifactController.java

```diff
@@ -46,18 +46,20 @@ public ArtifactController(ArtifactServices artifactServices) {
   public List<String> getVersions(
       @PathVariable("provider") String provider,
       @PathVariable("name") String name,
-      @RequestParam(value = "releaseStatus", required = false) String releaseStatus) {
+      @RequestParam(value = "releaseStatus", required = false) String releaseStatus,
+      @RequestParam(value = "type", required = false, defaultValue = "deb") String type) {
```

Comment:
I added this as a request param and not a path variable because it would cause a conflict with between the path patterns in `getArtifact` and `getVersions`.